hw4/02-sessions.js

```diff
@@ -1,17 +1,51 @@
 const express = require('express');
 const session = require('express-session');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 5001;
+// Use the express-session module
+// var sess;
+app.use(
+    session({
+        store: new session.MemoryStore(),
+        secret: 'keyboard cat',
+        resave: false,
+        saveUninitialized: true,
+        cookie: { secure: false },
+    })
+);
 
-// Add your code here
+// app.get('/', (req, res, next) => {
+//     // Add your code here
+//     //  sess.route = '';
+//     res.send('Hello');
+// });
```

Comment:
you can remove commented-out code. no need to commit I guess.

Reply:
will remove it.